lib/pages/signIn.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:flutter/material.dart';
+import '/colors.dart';
+
+class SignIn extends StatelessWidget {
+  const SignIn({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormState>();
+
+    return Scaffold(
+      // appBar: AppBar(),
+      body: Stack(
+        children: [
+          Positioned(
+            top: 0,
+            left: 0,
+            right: 0,
+            child: AppBar(
+              title: const Text('Rejestracja'),
+              centerTitle: true,
+            ),
+          ),
+          Center(
+              child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                RichText(
+                  text: const TextSpan(
+                    style: TextStyle(
+                        fontSize: 80,
+                        fontWeight: FontWeight.w800,
+                        color: Colors.black),
+                    children: [
+                      TextSpan(
+                          text: 'Para',
+                          style: TextStyle(color: AppColors.primaryColor)),
+                      TextSpan(
+                          text: 'gon',
+                          style: TextStyle(color: AppColors.defaultTextColor)),
+                    ],
+                  ),
+                ),
+                Container(
```

Comment:
This is very similar to 'Zaloguj się` from the main page. Consider extracting them